src/neo/SmartContract/Manifest/ContractAbi.cs

```diff
@@ -100,5 +100,14 @@ public JObject ToJson()
             json["events"] = new JArray(Events.Select(u => u.ToJson()).ToArray());
             return json;
         }
+
+        internal ContractAbi ForVersion(uint version)
```

Comment:
Why do we add this to `ContractAbi`? I think it is only related to native contracts.

Reply:
Moved